src/main/java/com/minecolonies/coremod/commands/citizencommands/AbstractCitizensCommands.java

```diff
@@ -179,5 +206,5 @@ private static int getValidCitizenId(final Colony colony, final boolean firstArg
      * @param colonyId  the id for the colony
      * @param citizenId the id for the citizen
      */
-    public abstract void executeSpecializedCode(@NotNull final MinecraftServer server, final ICommandSender sender, final Colony colonyId, final int citizenId);
+    public abstract void executeSpecializedCode(@NotNull MinecraftServer server, ICommandSender sender, Colony colonyId, int citizenId);
```

Comment:
why no finals?

Reply:
final got lost in the initial rewrite.  Added it back now.  Thanks for the catch.

Reply:
@raycoms No, actually, these were in there originally, but I got warnings for them.  I have no idea why they are considered redundant.  Same deal if I put them on the interface.   I don't care either way, so let me know what you prefer.

Description	Resource	Path	Location	Type
Redundant Modifier: Redundant 'final' modifier.	AbstractCitizensCommands.java	/minecolonies/src/main/java/com/minecolonies/coremod/commands/citizencommands	line 204	Checkstyle Problem
Redundant Modifier: Redundant 'final' modifier.	AbstractCitizensCommands.java	/minecolonies/src/main/java/com/minecolonies/coremod/commands/citizencommands	line 204	Checkstyle Problem

Reply:
I prefer with them, so the override adds them correctly as well =)